test/core/test_project_status.py

```diff
@@ -288,13 +288,13 @@ def test_get_exceeded_thresholds(self):
         testrun.metrics.create(name='metric1', suite=self.suite, result=5)
         status = ProjectStatus.create_or_update(build)
         MetricThreshold.objects.create(project=self.project,
+                                       environment=self.environment,
```

Comment:
I think there can have a test with two metricthresholds. This means two environments therefore two testruns and metrics. This way you can test that a metric exceeded in one environment but didn't for another environment.

Reply:
@chaws I've changed this to conform with what @terceiro explained about the threshold. Two cases, one with higher_better true then false.